python/plugins/processing/gui/ExtentSelectionPanel.py

```diff
@@ -205,10 +152,9 @@ def setValueFromRect(self, r):
         self.dialog.activateWindow()
 
     def getValue(self):
-        if unicode(self.leText.text()).strip() != '':
-            return unicode(self.leText.text())
-        else:
-            return self.getMinCoveringExtent()
+        if unicode(self.leText.text()).strip() == '':
+            return None
```

Comment:
On reflexion optional = True by default is a good workaround for the moment. I will set it soon.
Thanks for review.